egs/wsj/s5/steps/libs/tensorflow/tensorflow_nnet/nnetactivations.py

```diff
@@ -0,0 +1,133 @@
+# Copyright 2016    Vincent Renkens
```

Comment:
you'll have to check into the copyrights for these files... hopefully it's something compatible with Apache 2.0, or MIT license.  If not, he may agree to release under Apache 2.0.